src/ubx.cpp

```diff
@@ -1823,6 +1878,12 @@ GPSDriverUBX::payloadRxAddMonVer(const uint8_t b)
 
 			if (fwver_str != nullptr) {
 				GPS_INFO("u-blox firmware version: %s", fwver_str + strlen("FWVER="));
+
+				// Check if its a ZED-F9P-15B
+				if (strstr(fwver_str, "HPGL1L5")) {
```

Comment:
Is "HPGL1L5" going to be unique to this module or should we limit it to if hwVersion == "00190000" to be paranoid?

Reply:
See updated